src/code.rs

```diff
@@ -589,6 +651,33 @@ fn build_table_fns(
 
     buffer.push_str("}\n");
 
+    #[cfg(feature = "advanced-queries")]
+    // generate filter struct for filter() helper function
+    {
+        let filter_fields = table
```

Comment:
same here, this could also be done without a intermediate vec (though not necessary now)